Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -551,7 +551,7 @@ def bottle_formula(f, args:)
         },
         "bottle"  => {
           "root_url" => bottle.root_url,
-          "prefix"   => bottle.prefix,
+          "prefix"   => Homebrew::DEFAULT_PREFIX, # TODO: deprecate or parse this from cellar value
```

Comment:
Is this correct if you're bottling in a non-default prefix/cellar? I agree it should be ideal deprecated, though. Could you chuck a `3.3.0` reference in that comment somewhere?
```suggestion
          "prefix"   => Homebrew::DEFAULT_PREFIX, # TODO: 3.3.0 deprecate this
```